### DISCUSSION TITLE
Does UNO support Microsoft.UI.Windowing?

### DISCUSSION THREAD

**Discussion by MarkBrainfire** [unoplatform/uno#8338] [General]
WinUI 3 depends upon the WindowsAppSDK 1.0.
A key API within this SDK is Microsoft.UI.Windowing that provides a high-level, easy to use API for moving and resizing windows, 3 different Presenters: Overlapped, true Fullscreen, and PIP Compact Presenter, and dealing with multiple monitors (referred to as DisplayAreas).

Does UNO support translating this API cross platform?

**Comment by jeromelaban** [comment]
At this time, multi-window in Uno is not yet supported. You can track the progress here: https://github.com/unoplatform/uno/issues/8341
